### ISSUE TITLE
Forbidden Reach: Wrong gold times

### ISSUE THREAD

**Issue by Kepas-Beleglorn** [hasore/DragonRidingRace#1]
The supposed gold times default to 43s/42s/42s for every race.
The gold target times for those are:
```
Race                        Basic       Advanced    Reverse
Morqut Ascent: Gold         52.000 s    49.000 s    52.000 s
Forbidden Reach Rush: Gold  59.000 s    56.000 s    57.000 s
Southern Reach Route: Gold  70.000 s    68.000 s    63.000 s
Stormsunder Crater: Gold    43.000 s    42.000 s    42.000 s
Aerie Chasm: Gold           53.000 s    50.000 s    50.000 s
Caldera Coaster: Gold       58.000 s    52.000 s    49.000 s
```

**Comment by hasore** [comment]
Thank you.

**Comment by hasore** [comment]
Thank you, a fix will be available for release 0.2.2.

**Comment by hasore** [comment]
fixed.
